packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -43,8 +45,45 @@ export class TestRunnerService {
 		private readonly activeExecutions: ActiveExecutions,
 		private readonly testRunRepository: TestRunRepository,
 		private readonly testMetricRepository: TestMetricRepository,
+		private readonly nodeTypes: NodeTypes,
 	) {}
 
+	private getStartNodesData(
```

Comment:
Could we be more explicit about what data this returns, either by renaming the fn or adding a comment? Is it input data of the start nodes?

Reply:
I tried to add some context in the comment. This entire method is preparing a part of the large data structure that would be supplied to the workflow runner, to ensure the test execution will "re-play" the past execution from the same trigger node.